roles/openshift_node/tasks/main.yml

```diff
@@ -22,7 +22,7 @@
       deployment_type: "{{ openshift_deployment_type }}"
   - role: node
     local_facts:
-      labels: "{{ openshift_node_labels | default(none) }}"
+      labels: "{{ lookup('oo_option', 'openshift_node_labels') | default( openshift_node_labels | default() ) }}"
```

Comment:
@menren Looks good to me, but will need to be tested by @wshearn to verify it works for him.

Please update this PR with the change and when you're done, ask @wshearn to test it.

Thanks!